app/models/comment.rb

```diff
@@ -1,4 +1,4 @@
 class Comment < ApplicationRecord
   belongs_to :user
-  belongs_to :post
+  belongs_to :post, counter_cache: :comments_counter
 end
```

Comment:
It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_comments_counter method every time a new post has been saved to database, then make the update_comments_counter method private.

Reply:
Thanks for the updates and positive remarks @KayLemba, changes have been made to implement the above instructions.